### PR TITLE
disable istio mii tests for 12.2.1.3 images

### PR DESCRIPTION
**Pull request by vanajamukkara**
Jenkins - 

https://build.weblogick8s.org:8443/job/wko-kind-weekly-multiconfig/ - tests were failing on 12.2.1.3 images before, now they pass as istio mii tests are disabled for 12.2.1.3 images because of known issue.

latest jenkins - 
https://build.weblogick8s.org:8443/job/wko-kind-weekly-multiconfig/80/

### CODE REVIEW

**Review by anpanigr — COMMENTED** [review]
It will SKIP the tests with 12.2.1.3-CPU also. It should not skip in case of cpu image

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/TestConstants.java [hunk 1]
```diff
@@ -335,6 +335,7 @@ public interface TestConstants {
       "-Dweblogic.security.SSL.ignoreHostnameVerification=true -Dweblogic.security.TrustKeyStore=DemoTrust";
 
   public static final boolean WEBLOGIC_SLIM = WEBLOGIC_IMAGE_TAG.contains("slim");
+  public static final boolean WEBLOGIC_12213 = WEBLOGIC_IMAGE_TAG.contains("12.2.1.3");
```
Comment (anpanigr) [inline integration-tests/src/test/java/oracle/weblogic/kubernetes/TestConstants.java:338]: Add some comment why we are skipping  as follows  ..
"application deployment through REST interface issue and is included in WebLogic 12.2.1.3.0 CPU's"

**Review by anpanigr — COMMENTED** [review]
Put comment why we are skipping the tests

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/TestConstants.java (hunk 1 above)
Comment (vanajamukkara) [reply integration-tests/src/test/java/oracle/weblogic/kubernetes/TestConstants.java]: added comments in the extensions class as that's where we disable

**Review by maggiehe00 — APPROVED** [review]
lgtm

**Review by anpanigr — APPROVED** [review]

**Pull request merged**